lib/src/op_store.rs

```diff
@@ -390,6 +404,10 @@ impl Operation {
             view_id: root_view_id,
             parents: vec![],
             metadata,
+            // The root operation is guaranteed to have no new commits.
+            // TODO: Alternatively, the root commit could be "created" at the
+            // root operation?
```

Comment:
That seems inconsistent with other changes not being in the operation that creates them (they're only in the operation that rewrites them).

Reply:
Creation is recorded as `{new_commit_id: []}`, which helps terminate traversal of evolution history. What I'm not certain is whether the root commit is considered existent before the root operation, or newly created at the root operation.

Reply:
Thinking this a bit more. Since commits created/rewritten in abandoned ancestor operations aren't considered born in the root operation, it'll probably make sense that the root commit existed before the root operation as well.